litmus/director/openebs-upgrade-jiva/run_litmus_test.yml

```diff
@@ -2,7 +2,7 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  generateName: openebs-upgrade-jiva-
+  generateName: volume-upgrade-jiva-check-
   namespace: litmus
 spec:
   template:
```

Comment:
@amitbhatt818 
- Can you change the extra spaces in line-46? (run_litmus_test.yml)
- I think we are not using `test_name` variable anywhere please remove this ( from test_vars.yml ) or if we are using it then give value - `volume-upgrade-jiva-check` this has to be done for all other experiments by experiment owners.